docs/components/connectors/custom-built-connectors/best-practices.md

```diff
@@ -0,0 +1,182 @@
+---
+id: best-practices
+title: Best practices for custom-built element templates and connectors
+description: Learn the best practices for creating and managing custom-built element templates and connectors in Web Modeler.
+---
+
+## Best practices for creating element templates and connectors
```

Comment:
🔧 connector templates. This section is only concerned with templates afaik?

Reply:
yes, the correct title should be `for creating element and connectors templates` right?